tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/regression.py

```diff
@@ -73,11 +73,15 @@ def parameters(cls):
         params.update({
             'outputs': ListField(value_type=str, allow_empty=False, description='list of target output names')
         })
+        params.update({
```

Comment:
Also please add a description of the new parameter to adapters README: https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md